api/src/main/resources/metadata/Custom_Concepts.xml

```diff
@@ -5265,6 +5346,92 @@
     <concept_name  concept_name_id="161492"  concept_id="166157"  name="Follow Up Attempts"  locale="en"  locale_preferred="1"  creator="1"  date_created="2020-11-02 11:41:06"  concept_name_type="FULLY_SPECIFIED"  voided="0"    uuid="3e77064b-c43c-47c3-a140-0e8646928ec5"/>
     <concept_name  concept_name_id="161493"  concept_id="166158"  name="Missed Appointment Date"  locale="en"  locale_preferred="1"  creator="1"  date_created="2020-11-03 08:37:11"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="3aed47d1-a53a-4838-be7a-b51b7535a7f4"/>
 
+    <concept_name  concept_name_id="161496"  concept_id="166161"  name="Cardiac disease"  locale="en"  locale_preferred="1"  creator="1"  date_created="2021-02-01 16:45:34"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="9c0c8516-c58e-4ee1-86df-48e66f921cc9"  />
+    <concept_name  concept_name_id="161497"  concept_id="166162"  name="Previous illness"  locale="en"  locale_preferred="0"  creator="1"  date_created="2021-02-02 10:41:55"  concept_name_type="NULL"  voided="1"  voided_by="1"  date_voided="2021-02-02 11:38:03"  void_reason="Deleted due to change of name"  uuid="b7ff2435-6619-425a-be3b-6a452a604f43"  date_changed="2021-02-02 11:38:03"  changed_by="1"/>
```

Comment:
Why do you have a concept_name_type="NULL" you either leave it out or give it (FULLY_SPECIFIED,INDEX_TERM,SHORT).

Reply:
This is resolved